redis_store.go

```diff
@@ -178,7 +178,7 @@ func (this *RedisStore) Poll(c chan []byte, queue string) error {
 				continue
 			}
 
-			message, err := redis.Bytes(consumer.Do("LPOP", queue))
+			message, err := redis.Bytes(consumer.Do("BLPOP", queue, 10))
```

Comment:
seems like a reasonable change but might be better off in a separate PR depending on the author ofc